src/Transition/CMakeLists.txt

```diff
@@ -24,94 +22,27 @@ set(LIB_SRC
-list(APPEND VERSIONS 7_0_0)
-list(APPEND VERSIONS 7_1_0)
-list(APPEND VERSIONS 7_2_0)
-list(APPEND VERSIONS 8_0_0)
-list(APPEND VERSIONS 8_1_0)
-list(APPEND VERSIONS 8_2_0)
-list(APPEND VERSIONS 8_3_0)
-list(APPEND VERSIONS 8_4_0)
-list(APPEND VERSIONS 8_5_0)
-list(APPEND VERSIONS 8_6_0)
-list(APPEND VERSIONS 8_7_0)
-list(APPEND VERSIONS 8_8_0)
-list(APPEND VERSIONS 8_9_0)
-list(APPEND VERSIONS 9_0_0)
-list(APPEND VERSIONS 9_1_0)
-list(APPEND VERSIONS 9_2_0)
-list(APPEND VERSIONS 9_3_0)
-list(APPEND VERSIONS 9_4_0)
-list(APPEND VERSIONS 9_5_0)
-list(APPEND VERSIONS 9_6_0)
-
-if(APPLE)
-  if("${CMAKE_Fortran_COMPILER_ID}" MATCHES "GNU")
-    if(NOT "Ninja" STREQUAL ${CMAKE_GENERATOR})
-      set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -cpp")
-    endif()
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-none")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fbounds-check")
-  else()
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fpp")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -static-intel")
-  endif()
-elseif(UNIX)
-  set(CMAKE_SKIP_RPATH TRUE)
-  if("${CMAKE_Fortran_COMPILER_ID}" MATCHES "GNU")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -static")
-    if(NOT "Ninja" STREQUAL ${CMAKE_GENERATOR})
-      set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -cpp")
-    endif()
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-none")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fbounds-check")
-    set(STATIC_EXE TRUE)
-  else()
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fpp")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -static-intel")
-  endif()
-else() # Windows
-  set(STATIC_EXE TRUE)
-  if("${CMAKE_Fortran_COMPILER_ID}" MATCHES "GNU")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -static")
-    if(NOT "Ninja" STREQUAL ${CMAKE_GENERATOR})
-      set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -cpp")
-    endif()
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-none")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fbounds-check")
-  else()
-    # Set release flags to be empty
-    set(CMAKE_Fortran_FLAGS_RELEASE "")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} /libs:static")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} /fpp")
-  endif()
+if(NOT UNIX)  # Need to reinclude it on Windows
+  include("${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Fortran.cmake")
 endif()
-
+get_target_property(F_INTERFACE_COMPILE_FLAGS fortran_project_options INTERFACE_COMPILE_OPTIONS)
+message(DEBUG "Transition: F_INTERFACE_COMPILE_FLAGS=${F_INTERFACE_COMPILE_FLAGS}")
+target_link_libraries(TransitionLib PUBLIC fortran_project_options) # ffree-line-length-none
+
+set(VERSIONS
+    1_0_0  1_0_1  1_0_2  1_0_3  1_1_0  1_1_1  1_2_0  1_2_1  1_2_2  1_2_3  1_3_0  1_4_0
+    2_0_0  2_1_0  2_2_0  3_0_0  3_1_0  4_0_0  5_0_0  6_0_0  7_0_0  7_1_0  7_2_0
+    8_0_0  8_1_0  8_2_0  8_3_0  8_4_0  8_5_0  8_6_0  8_7_0  8_8_0  8_9_0
```

Comment:
I am compelled to want the 3's, 4, 5, 6, and 7's in their own lines....... :eyes: But I won't hold this up for it.